scripts/utils/cli-step.ts

```diff
@@ -5,9 +5,11 @@ import type { OptionSpecifier, OptionValues } from './options';
 import { createOptions, getCommand } from './options';
 
 const require = createRequire(import.meta.url);
-const cliExecutable = require.resolve('../../code/core/bin/index.cjs');
-const toolboxExecutable = require.resolve('../../code/lib/cli-storybook/bin/index.cjs');
-const createStorybookExecutable = require.resolve('../../code/lib/create-storybook/bin/index.cjs');
+const cliExecutable = require.resolve('../../code/core/dist/bin/dispatcher.js');
```

Comment:
Out of curiosity. Why can't `import.meta.resolve` be used here?

Reply:
it potentially can, but we're not focusing on migrating our scripts to ESM, except for the new scripts we write in `scripts/build`.